distribution/sdk/sample-nsili-server/src/main/java/org/codice/alliance/nsili/mockserver/server/MockNsili.java

```diff
@@ -70,6 +85,69 @@ public void startWebServer(int port) {
         sf.create();
     }
 
+    public void startFtpWebServer(int port) {
+        FtpServerFactory ftpServerFactory = new FtpServerFactory();
+
+        ListenerFactory listenerFactory = new ListenerFactory();
+        listenerFactory.setPort(port);
+        ftpServerFactory.addListener("default", listenerFactory.createListener());
+
+        PropertiesUserManagerFactory propertiesUserManagerFactory =
+                new PropertiesUserManagerFactory();
+        UserManager userManager = propertiesUserManagerFactory.createUserManager();
+        BaseUser baseUser = new BaseUser();
+        baseUser.setName("admin");
+        baseUser.setPassword("admin");
+
+        try {
+            ftpHomePath = Files.createTempDirectory("home_");
+            baseUser.setHomeDirectory(ftpHomePath.toString());
+            setDeleteTempDirectoryOnExit();
+        } catch (IOException e) {
+            System.out.println("ERROR: Unable to set ftp endpoint to a temporary home directory.");
+        }
+
+        try {
+            userManager.save(baseUser);
+            ftpServerFactory.setUserManager(userManager);
+
+            ConnectionConfigFactory connectionConfigFactory = new ConnectionConfigFactory();
+            connectionConfigFactory.setAnonymousLoginEnabled(true);
+            FtpServer ftpServer = ftpServerFactory.createServer();
+            ftpServer.start();
+
+        } catch (FtpException e) {
+            System.out.println("ERROR: Unable to start FTP server.");
+        }
+
+        System.out.println(
+                "INFO: Setting the ftp server's publish address to be ftp://localhost:" + port
+                        + "/");
+    }
+
+    private void setDeleteTempDirectoryOnExit() throws IOException {
+        Runtime.getRuntime()
```

Comment:
I think this could be simplified by setting deleteOnExit when creating the file https://docs.oracle.com/javase/7/docs/api/java/io/File.html#deleteOnExit()